engine/src/main/java/org/terasology/rendering/cameras/OculusStereoCamera.java

```diff
@@ -172,6 +172,9 @@ public Matrix4f getInverseViewProjectionMatrix() {
         return null;
     }
 
+    /**
+     * @deprecated
```

Comment:
It would be quite helpful to know why this is deprecated and what to use instead. If you don't know, you might want to leave this as is, so somebody who does might stumble over it in the future and comment this properly.